classes/01class/exercises/c01-e04/starkmatt/c01-e04.txt

```diff
@@ -0,0 +1,13 @@
+#Created IAM Role for S3FullAccess, went back into my Inital Public instance and changed IAM Rule to include S3FullAccess
```

Comment:
That looks  good  but what about specifying granular  permissions for read/write instead of using S3FullAccess. As  least privilege is not part of the exercise, I won't ask  for the change, but feel free to update that and submit another PR on how to achieve that if  you have some spare  time